src/plugins/intel_cpu/tests/functional/single_layer_tests/rdft.cpp

```diff
@@ -14,54 +14,86 @@ using namespace ov;
 
 namespace CPULayerTestsDefinitions {
 
-using RDFTTestCPUParams = std::tuple<
-        Shape,
-        std::vector<int64_t>,  // axes
-        std::vector<int64_t>,  // signal sizes
-        bool,                  // inverse
-        CPUSpecificParams>;
+using RDFTTestCPUParams = std::tuple<std::vector<InputShape>,
+                                     std::vector<int64_t>,  // axes
```

Comment:
Axes and Signal size inputs also can be dynamic. 
Also please add Precision parameter here. It can be bf16 or fp16 in future.

Reply:
> Axes and Signal size inputs also can be dynamic.

This is covered by the tests in this PR.

> Also please add Precision parameter here. It can be bf16 or fp16 in future.

Done

Reply:
Regarding the tests, they do not check dynamic axis or siganal shapes, only constant/non constant values. New shapes must be passed to the fn generate_inputs, as for the Data input.
The original comment is not only about the tests. The dynamic shapes support for axis and signal not implemented in the plugin as well. Also note that the non constant axes produce internal dynamism. Please look at the MVN node implementation. It also has internal dynamism.

Reply:
I made some changes to tests that run multiple inferences on dynamic model.
Regarding internal dynamism - why do you think it applies here? As far as I know it's when output shapes of the node can be calculated only after this node is executed, but that's not the case for RDFT.

Reply:
Internal dynamism is a case where the output shapes depend on the **input data**, **not** on the **input shapes**. The RDFT operation has output whose shape is calculated using axes and signal size values (not the shapes). So, if these inputs are not constant, the operation has internal dynamism.